src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://airbnb.com.us"
+    method="post"
+    class = "form"
+    data-qa="big"
+    >
+      <input class = "element"
+    type="text"
+    name = "big form"
+    placeholder="Try &quot;Los Angeles&quot;"
+    data-qa="keypress"
+    >
```

Comment:
Please apply code style recommendations from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
Check all over the page

```suggestion
    <form action="https://airbnb.com.us"
      method="post"
      class = "form"
      data-qa="big"
    >
      <input class = "element"
        type="text"
        name = "big form"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```